latest-release.ts

```diff
@@ -0,0 +1,31 @@
+import * as nodegit from "nodegit";
+import * as semver from "semver";
+
+
+export async function checkoutLatestRelease(
+    r: nodegit.Repository|Promise<nodegit.Repository>, dir?: string)
```

Comment:
Accepting a Promise as an argument seems weird. Why isn't this method just called in the .then() of the repo Promise?